src/main/java/org/folio/dcb/utils/KeyCloakTokenRequestHelper.java

```diff
@@ -0,0 +1,27 @@
+package org.folio.dcb.utils;
+
+import static java.util.Map.entry;
+
+import java.util.Map;
+
+import org.folio.dcb.integration.keycloak.model.DcbHubKCCredentials;
+
+import lombok.experimental.UtilityClass;
+
+@UtilityClass
+public class KeyCloakTokenRequestHelper {
```

Comment:
Up to you, but from my point of view - this method can be part of Keycloak service